### PR TITLE
New test created 

### PR DESCRIPTION
**Pull request by SAchu47**

### CODE REVIEW

File: test/articles.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      const res = await request
+        .post('/articles/')
+        .send({
+            title: 'iPhone 11',
+            description: 'A new dual‑camera system captures more of what you see and love. ',
+            author: 'Sachin',
+            tags: 'mobile'
+        })
+      expect(res.statusCode).toEqual(201)
+      expect(res.body).toHaveProperty('title')
```
Comment (jeffin143) [inline test/articles.test.js:28]:
```suggestion
      expect(res.body).toHaveProperty('title')
      done();
```

File: test/articles.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+// Connect to a new database before running any tests.
+beforeAll(async () => await dbHandler.connect());
+
+// Remove and close the db and server.
+afterAll(async () => await dbHandler.closeDatabase());
+
+describe('Article testing', () => {
+    
+    it('should create a new post', async () => {
```
Comment (jeffin143) [inline test/articles.test.js:18]:
```suggestion
    it('should create a new post', async done => {
```

File: test/articles.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            title: 'iPhone 11',
+            description: 'A new dual‑camera system captures more of what you see and love. ',
+            author: 'Sachin',
+            tags: 'mobile'
+        })
+      expect(res.statusCode).toEqual(201)
+      expect(res.body).toHaveProperty('title')
+    })
+
+    it('should get a post which is posted in pervious test', async () => {
```
Comment (jeffin143) [inline test/articles.test.js:31]:
```suggestion
    it('should get a post which is posted in pervious test', async done => {
```

File: test/articles.test.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        const res = await request
+        .get('/articles/')
+        expect(res.statusCode).toEqual(201)
+        expect(res.body[0].author).toEqual('Sachin')
```
Comment (jeffin143) [inline test/articles.test.js:35]:
```suggestion
        expect(res.body[0].author).toEqual('Sachin')
        done();
```

**Comment by jeffin143** [comment]
@SAchu47 , Made apt changes, Now this passes the CI

**Comment by SAchu47** [comment]
> @SAchu47 , Made apt changes, Now this passes the CI

Nice to see it working Thanks

**Comment by jeffin143** [comment]
Suggest you to name test/jest/test_name.js 

Because I am working on mocha testing do
It will have test/mocha/testname.js

In package.json
```
Test: jest test/jest/*.js
Test-mocha : mocha test/mocha/*.js
```

And you can run both test

**Comment by SAchu47** [comment]
> Suggest you to name test/jest/test_name.js
> 
> Because I am working on mocha testing do
> It will have test/mocha/testname.js
> 
> In package.json
> 
> ```
> Test: jest test/jest/*.js
> Test-mocha : mocha test/mocha/*.js
> ```
> 
> And you can run both test

Okay we would follow that

**Comment by jeffin143** [comment]
Please wait #8 , To be merged and then you can process on with testing functions along with jest

Let's avoid API testing with jest, We will continue with mocha for API and only test function / unit test with jest

Please refer https://github.com/khacksOSS/kapp-api/pull/3#issuecomment-619379650 
For further clarity 

cc @dinolinjob

**Comment by SAchu47** [comment]
I will recommit jest test after refactoring mixed up little for now. Will make new pr

**Pull request closed without merging**